### PR TITLE
Detailed post about v5

### PR DESCRIPTION
**Pull request by mikeal**

### CODE REVIEW

File: locale/en/blog/community/node-v5.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The other release line is called **Stable**. All release lines will be odd numbers, and have a shorter lifespan and more frequent updates to the code. The Stable release line will focus on active development of necessary features and refinement of existing APIs. Node.js version 5 is this type of release.
+
+We want to make sure that you are adopting the release that best meets your Node.js needs, so to break it down:
+
+Stay on or upgrade to Node.js v4.2.x if you need stability and have a complex production environment, e.g. you are a medium or large enterprise.
+
+Upgrade to Node.js v5.x if you have the ability to upgrade versions quickly and easily without distributing your environment.
```
Comment (Trott) [inline locale/en/blog/community/node-v5.md:22]: Typo? I think you mean `disturbing` rather than `distributing`?

Comment (Trott) [inline locale/en/blog/community/node-v5.md:22]: To fight having a Wall-Of-Text, this short paragraph and the previous one are good candidates to convert to bullets (especially given the colon ending the paragraph before that).

File: locale/en/blog/community/node-v5.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Now that you have the very basics, let’s take a deeper look at the new features and characteristics of v5, and the benefits and details of our LTS plan.
+
+## Introduction to Node.js v5
+
+[Node.js v5](https://nodejs.org/en/blog/release/v5.0.0/) is an intermediate feature release line that is best suited for users who have an easier time upgrading their Node.js installations, such as developers using the technology for front-end toolchains. This version will be supported for a maximum of only eight months and will be continually updated with new features and better performance; it is not supported under our LTS plan.
+
+The release cadence for v5.x will be more rapid than in the past. Expect a new release once every one to two weeks for v5.x. If upgrading is a challenge for you, we suggest you do not use this release. There will be significant ongoing development and the focus is on trying to get the releases into users’ hands as soon as it is practical do so.
```
Comment (Trott) [inline locale/en/blog/community/node-v5.md:30]: Typo: `practical to do so`. I'd recommend tightening up the whole sentence (and splitting it into two sentences), though: `There will be significant ongoing development. The focus is on getting the releases to users' as soon as possible.`

**Comment by mikeal** [comment]
@trott because this was already merged I committed your suggestions to master.

**Pull request merged**